edxval/tests/test_serializers.py

```diff
@@ -98,8 +98,8 @@ def test_invalid_course_id(self):
         )
         self.assertFalse(serializer.is_valid())
         self.assertEqual(
-            serializer.errors,
-            {"courses": ["Ensure this value has at most 255 characters (it has 300)."]}
+            str(serializer.errors['courses']['course_id'][0]),
```

Comment:
This seems like a different error structure than previously.  Did this change because of an upstream requirment changing or somethnig else?

Reply:
This seems to be related to inadvertently upgrading django-rest-framework, which changes the error message representation. I'll revert this.